specs/agents/tracing-distributed-tracing.md

```diff
@@ -146,6 +146,11 @@ If an incoming request contains either of the `traceparent` or `tracestate`
 headers, they should be propagated throughout the transaction and mutated as
 specified above before being set on outgoing requests.
 
+There is an edge case where `tracestate` is present but blank `""` - in this
+case, we should not propagate the tracestate header. Propagating the header in
+this case can cause downstream receivers of requests from the monitored application
+to error since an empty header is invalid.
```

Comment:
According to https://github.com/elastic/apm-agent-rum-js/issues/1232#issuecomment-1143855956, this doesn't seem to be entirely correct

> As to the HTTP spec, my read of the ABNF at https://datatracker.ietf.org/doc/html/rfc7230#section-3.2 is that an empty header value is explicitly allowed. However, this is about supporting picky proxies/servers that do not like empty header values.

Reply:
Ah, very true, let me be clearer 👍